integr8ly/cloud-resource-operator
# update cluster vpc setup to use /28 cidr subnet mask

currently we are using a 20 bit subnet mask for the cidr block.
however, there is no need for the subnet mask to be so large.

this change updates the subnet mask to instead be 28 bits in the
cidr block, this is the aws minimum possible value and should
avoid clashes with openshift clusters provisioned in vpcs with a
cidr block with a mask smaller than 21 bits.

verification (fresh install):
- from this commit, create a postgres and redis resource using
  the aws provider
- ensure the subnet created for rds and elasticache now use a 28
  bit subnet mask
- ensure the creation for both rds and elasticache resources are
  created successfully

verification (backwards compatible):
- from master, create a postgres and redis resource using the
  aws provider
- from this commit, run the operator, ensure the old subnets
  are used
- ensure the creation for both rds and elasticache resources are
  created successfully

verification (rhmi):
- perform the previous two verification blocks, with a full rhmi
  installation instead of a single resource.
- run rhmi functional test suite against the completed cluster
